tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -811,7 +782,6 @@ struct RTFStreamerInfoObject {
                   'n', 't', 'a', 'l', ':', ':', 'R', 'N', 'T', 'u', 'p', 'l', 'e'};
    char fLTitle = 0;
 
-   RInt32BE fChecksum{ChecksumRNTupleClass()};
```

Comment:
Are you sure we can just remove that member? If we write out the object with RMiniFile, TFile can read it back correctly?